reference/tools/ticdc/sink.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Configure Sink URI
+category: reference
+---
+
+# Configure Sink URI
+
+Configure sink UIR in the following format. Currently, `scheme` supports `mysql`, `tidb`, and `kafka`.
+
+{{< copyable "" >}}
+
+```
+[scheme]://[userinfo@][host]:[port][/path]?[query_parameters]
+```
+
+## Configure sink URI with `mysql` or `tidb`
+
+The following command is a configuration example:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+--sink-uri="mysql://root:123456@127.0.0.1:3306/?worker-count=16&max-txn-row=5000"
+```
+
+The parameters in the above command are described as follows:
+
+| Parameter         | Description                                             |
+| :------------ | :------------------------------------------------ |
+| `root`        | The name of the downstream database                             |
+| `123456`       |  The password of the downstream database                                   |
+| `127.0.0.1`    |  The IP of the downstream database                              |
+| `3306`         |  The port of the downstream database                               |
+| `worker-count` |   The SQL statement count that can be concurrently executed downstream (optional, `16` by default)    |
+| `max-txn-row`  |  The size of transaction batches that can be executed downstream (optional, `256` by default)  |
+
+## Configure sink URI with `kakfa`
```

Comment:
```suggestion
## Configure sink URI with `kafka`
```
Typo in Chinese, plz fix it in https://github.com/pingcap/docs-cn/pull/2493/ @TomShawn

Reply:
Fixed.